buildSrc/src/main/groovy/com/github/okbuilds/okbuck/composer/JavaTestRuleComposer.groovy

```diff
@@ -0,0 +1,69 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2015 Piasy
```

Comment:
We should consider changing all the license infos to point to OkBuilds at some point as well @Piasy